app/src/main/java/com/example/parkingsystem/mvp/view/base/FragmentView.java

```diff
@@ -0,0 +1,23 @@
+package com.example.parkingsystem.mvp.view.base;
+
+import android.content.Context;
+import android.icu.util.Freezable;
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.Fragment;
+import java.lang.ref.WeakReference;
+
+public class FragmentView {
+    private WeakReference<DialogFragment> fragmentRef;
```

Comment:
This should be Fragment not DialogFragment

Reply:
Changed!